.github/workflows/builder.yml

```
@@ -43,4 +47,4 @@ jobs:
     - name: Build
       run: |
         npm ci
-        npm run build --if-present
+        npm run build-prod --if-present
```

Shall we rename the command to `build:prod`?
```suggestion
        npm run build:prod --if-present
```